subprojects/packagefiles/tree-sitter/meson.build

```diff
@@ -0,0 +1,29 @@
+project('tree-sitter', 'c',
+  version: '0.15.7',
+  license: 'MIT',
+  meson_version: '>=0.46.0',
+)
+
+tree_sitter = library('tree-sitter', 'lib/src/lib.c',
+  include_directories: [
+    include_directories('lib/src'),
+    include_directories('lib/include'),
+    include_directories('lib/utf8proc')
+  ],
+  soname: '0.0',
+  install:true,
+)
+
+install_headers('lib/include')
```

Comment:
I haven't looked, but does this need to be under a subdir?

Also does install_headers() actually work like this?

Reply:
It doesn't, and once the CI error in releases.json is fixed the CI will actually run meson and provide error messages to the effect that this isn't a file on disk.